packages/react-devtools-shared/src/devtools/views/Components/HooksTree.js

```diff
@@ -202,17 +202,11 @@ function HookView({canEditHooks, hook, id, inspectPath, path}: HookViewProps) {
               className={name !== '' ? styles.Name : styles.NameAnonymous}>
               {name || 'Anonymous'}
             </span>
+            <span className={styles.Value} onClick={toggleIsOpen}>
```

Comment:
This should only be shown if the hook is collapsed. Otherwise, the `<KeyValue>` component should be rendered as it enables the complex value to be expanded and inspected.

So something like this:
```js
<div className={styles.Hook}>
  <div ref={contextMenuTriggerRef} className={styles.NameValueRow}>
    <ExpandCollapseToggle isOpen={isOpen} setIsOpen={setIsOpen} />
    <span
      onClick={toggleIsOpen}
      className={name !== '' ? styles.Name : styles.NameAnonymous}>
      {name || 'Anonymous'}
    </span>
    <span className={styles.Value} onClick={toggleIsOpen}>
      {isOpen || getMetaValueLabel(value)}
    </span>
  </div>
  <div className={styles.Children} hidden={!isOpen}>
    <KeyValue 
      depth={1} 
      alphaSort={false} 
      inspectPath={inspectPath} 
      name="DebugValue" 
      path={path.concat(['value'])} 
      pathRoot="hooks"  
      value={value} 
    />
    {subHooksView}
  </div>
</div>
```

Reply:
To be clear, the label should work like this when toggling:
![Custom hook label](https://user-images.githubusercontent.com/29597/76897744-8b7baf80-6851-11ea-94ec-ef37e02259ed.gif)

Reply:
Yep that's what I wanted. I guess in this PR the debug value is still displayed on the same line as "CustomObject:" when expanded?

Reply:
Yeah, the string preview just stayed visible but the ability to expand the complex value below was gone. I added it back in 32bac9e :smile: